<launch>
  <node pkg="sonar" name="sonar_node" type="sonar_node" />
  <node pkg="sonar" name="data_handle" type="data_handle" />

  <node pkg="hokuyo_node" name="hokuyo" type="hokuyo_node" respawn="false" output="screen">
    <param name="calibrate_time" type="bool" value="false"/>
    <param name="port" type="string" value="/dev/ttyACM0"/>
    <param name="intensity" type="bool" value="false"/>
  </node>  
</launch>

